code/modules/hydroponics/grown/cocoa_vanilla.dm

```diff
@@ -50,3 +50,47 @@
 	foodtype = FRUIT
 	tastes = list("vanilla" = 1)
 	distill_reagent = /datum/reagent/consumable/vanilla //Takes longer, but you can get even more vanilla from it.
+
+/obj/item/seeds/cocoapod/bungotree
+	name = "pack of bungo tree seeds"
+	desc = "These seeds grow into bungo trees. They look like they might contain a horrible cardiotoxin."
```

Comment:
how does a plant look like it might contain a horrible cardiotoxin?

Reply:
I just thought it was funny, it is based on the cocoa tree description. I will change it.